static/src/stylesheets/module/content/_media-player.scss

```diff
@@ -129,17 +134,12 @@ $ima-controls-height: 70px;
             top: 1.85em;
             border-style: solid;
             border-width: 1em 0 1em 2.4em;
-            border-color: transparent transparent transparent $neutral-7;
+            border-color: transparent transparent transparent #333333;
```

Comment:
this should be a colour variable https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/palette/_src.scss#L9